nav2_waypoint_follower/src/waypoint_follower.cpp

```diff
@@ -175,7 +223,7 @@ WaypointFollower::followWaypoints()
 
   while (rclcpp::ok()) {
     // Check if asked to stop processing action
-    if (action_server_->is_cancel_requested()) {
+    if (action_server->is_cancel_requested()) {
```

Comment:
Should L231 have `action_server` instead of `action_server_`. I believe the way it's written it would only ever `terminate_al()` for the "regular" `FollowWaypoints` action, and not for the `FollowGpsWaypoints` action.

Reply:
Wow, I think you are right! thanks for pointing this out